main.rb

```diff
@@ -0,0 +1,130 @@
+require_relative './book'
+require_relative './student'
+require_relative './teacher'
+require_relative './classroom'
+require_relative './rental'
+
+class App
+  def initialize
+    puts teacher.id
+    @books = []
```

Comment:
- [ ] Please your app is not running from my end, checkout the error message the image below and kindly fix the issue.